src/tap.test.ts

```diff
@@ -0,0 +1,69 @@
+import { filter } from './filter';
+import { map } from './map';
+import { pipe } from './pipe';
+import { tap } from './tap';
+
+const value = [1] as const;
```

Comment:
I prefer the name data, regardless, this is a const so have it ALL_CAPS

```suggestion
const DATA = [1] as const;
```

Reply:
:+1: Done